src/App/TaskListExecutionPage.xaml.cs

```diff
@@ -34,6 +34,7 @@ public TaskListExecutionPage()
             this.NavigationCacheMode = NavigationCacheMode.Enabled;
             _selectedTaskList = -1;
             _selectedTaskListGuid = Guid.Empty;
+            _selectedTask = -1;
```

Comment:
I dont see any code that ever modifies _selectedTask. It remains -1 forever.

Reply:
Noted, added the fix to consider the variable before making any selected index changes.